core/domain/email_manager_test.py

```diff
@@ -4920,6 +4921,280 @@ def test_email_sent_to_multiple_admins(self) -> None:
             self.CURRICULUM_ADMIN_2_EMAIL)
 
 
+class NotifyReviewersNewSuggestionsTests(
+    test_utils.EmailTestBase):
+    """Tests the send_mail_to_notify_contributor_dashboard_reviewers method,
+    which sends an email to reviewers with information regarding the suggestions
+    that have waited the longest for review.
+    """
+
+    target_id: str = 'exp1'
+    skill_id: str = 'skill_123456'
+    mocked_review_submission_datetime: datetime.datetime = (
```

Comment:
Can you please create datetimes with timezones? Naive datetimes can lead to hard to debug errors as times are converted into individual users' timezones.

Can you please make similar changes whereever you are creating naive datetimes within this PR

Reply:
Done, Thanks!